collector/src/main/java/com/navercorp/pinpoint/collector/service/TraceService.java

```diff
@@ -177,12 +191,21 @@ private void insertSpanEventStat(SpanBo span) {
     }
 
     private void insertSpanEventList(List<SpanEventBo> spanEventList, ServiceType applicationServiceType, String applicationId, String agentId, String endPoint) {
+
         for (SpanEventBo spanEvent : spanEventList) {
             final ServiceType spanEventType = registry.findServiceType(spanEvent.getServiceType());
+
+            if (spanEventType.isAlias()) {
+                insertAcceptorHost(spanEvent, applicationId, applicationServiceType);
```

Comment:
I think it would be safer to add a `continue;` here.
Even if the `if (!spanEventType.isRecordStatistics())` check is right below, but I think adding an explicit `continue;` here would safeguard against malformed cases where service type has both `AS_ALIAS` and `RECORD_STATISTICS`.
What do you think?

Reply:
There is a reason why I didn't add `continue;` there.

Since restraints are set in `TraceMetadataLoader.java`.
Logically, `continue;` doesn't have any effects.

Rather, it may be more reasonable to think that it should be included because it doesn't matter.

But I thought it would be a good idea to put the constraints together, not scattered across the codes. So If the day comes to remove those constraints. we can just take a look at the loading part.

What do you think?

Reply:
Sounds fair enough~